README.md

```diff
@@ -1,2 +1,24 @@
 # CheckPhoneSize
-Comparison check phone free size method
+Comparison check phone free size method.
+
+Device tested are in the following belows:
```

Comment:
Change to "Some of the devices that have been tested:"

Reply:
done 👍 .